src/ray/stats/metric_defs.h

```diff
@@ -0,0 +1,29 @@
+#ifndef RAY_METRIC_DEFS_H_
+#define RAY_METRIC_DEFS_H_
+
+/// Use `RAY_METRIC(name, description, unit)` to define a metric
+/// which parameters are `metric_name`, `description` and `unit`.
+
+static Metric TaskElapse = Metric::MakeHistogram("task_elapse",
+                                                 "the task elapse in raylet",
+                                                 "ms",
+                                                 {0, 25, 50, 75, 100},
+                                                 {NodeAddressKey, CustomKey});
```

Comment:
It's unclear to me what this metric exactly means. It measures the latency of which period of time?

Also, do we want to integrate these metrics to code in this PR? I think that'd be nicer.